gradle.properties

```diff
@@ -20,12 +20,12 @@ android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 
-version=0.3.2
+version=0.3.3-SNAPSHOT
```

Comment:
I am not sure about adding -SNAPSHOT to the version. I guess it can ease the development.

Reply:
I added SNAPSHOT so that it can get published to the local mvn repo and be used as a dependency in nessus-didcomm. With maven you cannot have code changes to an already released version - I guess that's a general truth.